content/blog/policy-attachments.md

```diff
@@ -60,15 +60,15 @@ spec:
         response_flags: "%RESPONSE_FLAGS%"
         bytes_received: "%BYTES_RECEIVED%"
 ```
-Applying policies to a gateway is handy as it lets you configure all traffic and all the routes that the Gateway serves. What if in addition to configuring a gateway, you also want to extend a route configuration? Kgateway’s[RoutePolicy](https://kgateway.dev/docs/reference/api/top-level/routepolicies/) API is designed to solve this use case for you.
+Applying policies to a gateway is handy as it lets you configure all traffic and all the routes that the Gateway serves. What if in addition to configuring a gateway, you also want to extend a route configuration? Kgateway’s[TrafficPolicy](https://kgateway.dev/docs/reference/api/top-level/TrafficPolicies/) API is designed to solve this use case for you.
 
-The structure of a RoutePolicy is very similar to the one of an HTTPListenerPolicy, but you use the `targetRef` section to apply the policy to the routes in an HTTPRoute instead of a Gateway. 
+The structure of a TrafficPolicy is very similar to the one of an HTTPListenerPolicy, but you use the `targetRef` section to apply the policy to the routes in an HTTPRoute instead of a Gateway. 
```

Comment:
```suggestion
The structure of a TrafficPolicy is very similar to the one of an HTTPListenerPolicy, but you use the `targetRef` section to apply the policy to the routes in an HTTPRoute instead of a Gateway. 
```

Reply:
could you maybe do another swipe for targetRefs?